docs/docs/widgets/statistics.md

```diff
@@ -39,6 +40,7 @@ This property lets you add and secondary sign either positive or negative,can be
 ### Show on desktop
 
 Toggle on or off to display the widget in desktop view. You can programmatically determing the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}`.
+
 ### Show on mobile
 
 Toggle on or off to display the widget in mobile view. You can programmatically determing the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}`.
```

Comment:
Can you please fix this typo `determing` and make it `determine` in `Show on desktop` and `Show on mobile` section?

Reply:
Done!